tests/tickets/gh663/manual/1.md

```diff
@@ -0,0 +1,11 @@
+@bender-tags: 4.7.1, bug, 663
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, widget, enterkey
+
+----
+
+1. Open developer console
```

Comment:
You can simplify this test to two steps:

1. Open developer console.
1. Click on scrollbar.

And add larger amount of content to editor so it is scrollable on init.